CaptureSDK/GiniCaptureSDK/Documentation/Sections/Documentation.md

```diff
@@ -1,21 +1,2 @@
-<p align="center">
-<img src="img/logo.png" width="250">
-</p>
-
-# Gini Capture SDK for iOS
-
-[![Platform](https://img.shields.io/badge/platform-iOS-lightgrey.svg)]()
-[![Devices](https://img.shields.io/badge/devices-iPhone%20%7C%20iPad-blue.svg)]()
-[![Swift version](https://img.shields.io/badge/swift-5.0-orange.svg)]()
-[![Swift package manager](https://img.shields.io/badge/Swift_Package_Manager-compatible-orange?style=flat-square)]()
-
-
-The Gini Capture SDK provides components for capturing, reviewing and analyzing photos of invoices and remittance slips.
-
-By integrating this SDK into your application you can allow your users to easily take a picture of a document, review it and getting analysis results from the Gini backend.
-
-We provide pre-defined screens that can be customized in a limited way. The screen and configuration design is based on our long-lasting experience with integration in customer apps.
-
-On *iPhone*, the Gini Capture SDK has been designed for portrait orientation.
-On *iPad* we support portrait and landscape orientations.
 
+# Gini Capture SDK Reference documentation
```

Comment:
[clarification] This file will remain empty?

Reply:
yes. Jazzy needs one page as a starting point for `index.html`.

Reply:
Thanks for telling me!